index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+$nazevParty = "Czechitas";
+$poradatel = "Ondra, Petr, Jirka";
+$misto = "Praha";
+$kapacita = 30;
+
+$ucastnici = explode("\n", file_get_contents("ucastnici.txt"));
+
+if (isset($_GET["pridej"])) {
+    $novyUcastnik = $_GET["pridej"];
+    $ucastnici[] = $novyUcastnik;
+
+    file_put_contents("ucastnici.txt", implode("\n", $ucastnici));
+
+    header("Location: /");
+    exit;
+}
+
+$pocetUcastniku = count($ucastnici);
+
+$volnaKapacita = $kapacita - $pocetUcastniku;
+
+if ($pocetUcastniku == 0) {
+    $pocetUcastniku = "Nikdo nepřijde :-(";
```

Comment:
```suggestion
    $pocetUcastniku = "Nikdo nepřijde 😞";
```

Reply:
tohle si tam klidne uprav sam :D ja uz to rebasovat/predelavt nechci :D

Reply:
Stejne budes